src/test/java/com/backbase/billpay/fiserv/payees/PayeesControllerTest.java

```diff
@@ -219,22 +221,25 @@ public void getBillPayPayeeById() throws Exception {
         
         List<PaymentService> paymentServices = payee.getPaymentServices();
         PaymentService regularPaymentService = paymentServices.get(0);
-        assertEquals(payeeSummary.getCutoffTime(), regularPaymentService.getCutoffTime());
+        assertEquals(toZonedDateTime(payeeSummary.getCutoffTime()), regularPaymentService.getCutoffTime()
+            .withZoneSameInstant(ZoneId.of("America/New_York")));
```

Comment:
Can you please make the ZoneId a constant here to be reused throughout the class?